pkg/service/v0/service.go

```diff
@@ -26,7 +30,7 @@ func NewService(opts ...Option) Service {
 	}
 
 	m.Route(options.Config.HTTP.Root, func(r chi.Router) {
-		r.Get("/", svc.Dummy)
+		r.Get("/remote.php/dav/files/{user}/*", svc.Thumbnail)
```

Comment:
Maybe I'm getting this wrong - but this will grab all requests to the files endpoint - which is the regular files endpoint.
Thumbnails endpoint is only executed if the query parameter 'preview' is in the request url - see https://github.com/owncloud/core/blob/2074883c3f66e9aa7d70e7522719a637c3161053/apps/dav/lib/Files/PreviewPlugin.php#L87

Reply:
Yes, that is true but since all other webdav code is currently in ocis-reva we don't need to worry about that right now. The ocis-webdav service will listen on another port.
We have to change that though when we move the webdav code from reva to this repo.

Reply:
well - why not build it correctly right away?

Reply:
Because then I would need to make assumptions about how the structure would be after we moved the webdav code in here. I doubt that the structure will stay the same.
Also changing it later will not be too dificult or complex so it seemed like a good tradeoff to have something quickly now.

Reply:
I see that we have different approaches to solve problems ;-)
Have fun! :wave: